main.go

```diff
@@ -1,8 +1,13 @@
 package main
 
-import "os"
+import (
+	"os"
+
+	"github.com/hashicorp/consul-terraform-sync/event"
+)
 
 func main() {
-	cli := NewCLI(os.Stdout, os.Stderr)
+	store := event.NewStore()
```

Comment:
The intent of why the store is initialized at this level isn't clear to me, could you expand on this?

Reply:
Yes! Sorry, it's a bit premature. I am planning to initialize the API in the main in the next PR, something like `NewAPI(store)` so that the status API could access the store. Let me know what you think!

Reply:
Gotcha. Ok it'll likely require some refactoring since the configs are parsed within `NewCLI` presuming `NewAPI` will need that info too

Reply:
Yes! Exactly as you say. I'll have to figure out how to share the configs across the cli and api and am planning to do that in the next PR. Sorry for the confusion of making early moves in adding store at this level!

Reply:
🤔 I thought about your question some more. I like the neatness of explicitly creating and executing CLI and API in the main. But sharing the config stuff will be messier than if the cli calls NewAPI. I think I'll try both ways and see how they look. Let me know if you have thoughts!